src/test/moment/locale.js

```diff
@@ -152,6 +152,10 @@ test('defineLocale', function (assert) {
     assert.equal(moment().locale('dude').locale(), 'dude', 'defineLocale defines a locale');
 });
 
+test('locales', function (assert) {
+    assert.equal(moment.locales().length > 0, true, 'locales returns an array of defined locales');
```

Comment:
It would be nice to see a smarter test here.  Test for content, at least for Wnglish, and perhaps include one custom locale (like `dude` shown in the test above)